test/common/alarm/AlarmFactory.test.ts

```diff
@@ -193,9 +193,9 @@ test("addAlarm: period override is propagated to alarm metric", () => {
     alarmNameSuffix: "TwoHoursPeriod",
     period: Duration.hours(2),
   });
-  const alarm1hConfig = alarm1h.alarm.metric.toMetricConfig();
+  const alarm1hConfig = (alarm1h.alarm as any).metric.toMetricConfig();
```

Comment:
Could cast as an `Alarm` for better type checking?

Reply:
Good catch, will change.